src/e2e-test/resources/BigQuery/BigQueryInsertDataQuery.txt

```diff
@@ -1,5 +1,5 @@
-insert into `DATASET.TABLE_NAME` (Name, Col_Timestamp__c, Col_Date__c, Col_Currency__c, Col_Email__c, Col_Number__c,
-Col_GeoLocation__Latitude__s, Col_GeoLocation__Longitude__s, Col__c, Col_Url__c, Col_Time__c, Col_Text__c) values
-('adam','2019-03-10 04:50:01 UTC','2021-01-28',61.823765812,'skfdsfds@gmail.com',898365444,37.794116,-122.3432,
+insert into `DATASET.TABLE_NAME` (Id__c, Name, Col_Timestamp__c, Col_Date__c, Col_Currency__c, Col_Email__c, Col_Number__c,
```

Comment:
Is this change done to fix the broken environment dependent tests?

Reply:
Yes, we have done this change to fix the broken environment dependent tests.